client/user/src/mealorder/meal-order-main-app.html

```diff
@@ -0,0 +1,357 @@
+                                    fallback-selection="view404"
+                                    role="main">
+
+                                <div name="menu">
+                                    <div class="history-of-orders"><a href="/#/history/">История заказов</a></div>
+                                    <div class="icon-container">
+                                        <a href="/#/history/">
+                                            <iron-icon class="history-icon" icon="maps:icons:history"></iron-icon>
+                                        </a>
+                                        <div>
+                                            <iron-icon class="order" icon="icons:shopping-cart"
+                                                       drawer-toggle></iron-icon>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div name="history">
+                                    <div class="history-of-orders menu-history"><a href="/#/menu/">Сделать заказ</a>
+                                    </div>
+                                    <div class="icon-container">
+                                        <a href="/#/menu/">
+                                            <iron-icon class="history-icon" icon="maps:icons:create"></iron-icon>
+                                        </a>
+                                        <div>
+                                            <iron-icon class="order" icon="icons:date-range"
+                                                       drawer-toggle></iron-icon>
+                                        </div>
+                                    </div>
+                                </div>
+                            </iron-pages>
+                        </div>
+                    </app-toolbar>
+                </app-header>
+
+                <iron-pages
+                        selected="[[routeData.page]]"
+                        attr-for-selected="name"
+                        fallback-selection="view404"
+                        role="main">
+                    <menu-order></menu-order>
+                    <user-history></user-history>
+                </iron-pages>
+
+            </app-header-layout>
+
+        </app-drawer-layout>
+    </template>
+    <script>
+        /**
+         * Main user meal order component.
+         *
+         * <p>Is adopted for both desktop and mobile.
+         * Consists of 2 page:
+         * 1. A page with vendor menus, and user's shopping cart.
+         * 2. A page with user's history.
+         *
+         * The redux mixin provides its own data system and data flow.
+         * Fires following redux events:
+         * <li>addDish</li>
+         * <li>removeDish</li>
+         * <li>changeHistoryDate</li>
+         *
+         * @customElement
+         * @polymer
+         * @ReduxMixin
```

Comment:
What does it mean? See https://www.polymer-project.org/2.0/docs/tools/documentation#class-mixins